ui/app/components/policy-editor.hbs

```diff
@@ -55,12 +54,12 @@
 
 	<footer>
 		{{#if (can "update policy")}}
-			<button
-				class="button is-primary"
-				type="submit"
-			>
-				Save Policy
-			</button>
+		<Hds::Button
+			@text="Save Policy"
+			@type="submit"
+			data-test-save-policy
+			{{on "click" this.save}}
+		/>
```

Comment:
A major part of this PR is adding new elements from our [official design system](https://helios.hashicorp.design/) —these Hds::Buttons are just some of the many throughout.